gist/gist.py

```diff
@@ -138,19 +138,6 @@ def __init__(self, token, editor=None):
         """
         self.token = token
         self.editor = editor
-        self.proxies = {}
```

Comment:
Ok, I have convinced myself that this change should not introduce a regression and that it is the right change to make.